src/main/java/com/powsybl/openloadflow/network/LfNetwork.java

```diff
@@ -75,6 +77,14 @@ public class LfNetwork extends AbstractPropertyBag implements PropertyBag {
 
     private GraphConnectivity<LfBus, LfBranch> connectivity;
 
+    private Graph<LfBus, LfBranch> dcZeroImpedanceSubGraph;
```

Comment:
Maybe zeroImpedanceSubGraph and  zeroImpedanceSpanningTree could be grouped in another object.

Reply:
Yes, it could be a good idea.